R/test_functions.R

```diff
@@ -0,0 +1,377 @@
+
+test_no_error <- function(object, info) {
```

Comment:
We've dropped the `expect`. For all there's, I'd prefer `test_expect_...`

Reply:
Changed all `test_` to `test_expect_...`